### DISCUSSION TITLE
如今的版本是否可以像官方版本一样进行剧情而不是使用代码来手动调取

### DISCUSSION THREAD

**Discussion by qq2928642231** [Melledy/Grasscutter#1988] [Q&A]
如今的版本是否可以像官方版本一样游玩剧情而不是使用代码来手动调取

**Comment by Simplxss** [comment]
有一个人的folk修了任务 但是仍然存在许多bug  你如果非常想看剧情的话可以去看看

**Reply by sadwwwwwqsad** [reply]
> 有一个人的folk修了任务 但是仍然存在许多bug 你如果非常想看剧情的话可以去看看

可否给个链接或者关键字
